pkg/builder/local_build_executor.go

```diff
@@ -248,6 +253,24 @@ func (be *localBuildExecutor) Execute(ctx context.Context, filePool re_filesyste
 		return response
 	}
 
+	if len(be.inputRootCharacterDevices) > 0 {
+		if err := inputRootDirectory.Mkdir("dev", 0777); err != nil && !os.IsExist(err) {
+			attachErrorToExecuteResponse(response, util.StatusWrap(err, "Unable to create dev directory in input root"))
+		}
+		devDir, err := inputRootDirectory.EnterDirectory("dev")
+		if err != nil {
+			attachErrorToExecuteResponse(response, util.StatusWrap(err, "Unable to enter dev directory in input root"))
+		}
+		for name, number := range be.inputRootCharacterDevices {
+			if _, err := devDir.Lstat(name); err == nil {
```

Comment:
Is this check needed?

Reply:
I think if that isn't there and the file already exists the Mknod call will return an error.

Reply:
But that's not necessarily a bad thing, right?